src/probnum/quad/_bayesquad.py

```diff
@@ -30,11 +30,13 @@ def bayesquad(
     domain: Optional[DomainLike] = None,
     measure: Optional[IntegrationMeasure] = None,
     policy: Optional[str] = "bmc",
+    scale_estimator: Optional[str] = "mle",
     max_evals: Optional[IntLike] = None,
     var_tol: Optional[FloatLike] = None,
     rel_tol: Optional[FloatLike] = None,
     batch_size: Optional[IntLike] = 1,
     rng: Optional[np.random.Generator] = np.random.default_rng(),
+    jitter: Optional[FloatLike] = 1.0e-6,
```

Comment:
If `jitter` is never `None`, then use 

```suggestion
    jitter: FloatLike = 1.0e-6,
```